src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1690,15 +1742,19 @@
 
 helps['network vnet check-ip-address'] = """
     type: command
-    short-summary: Check if a private IP address is available for use.
+    short-summary: Check if a private IP address is available for use within a virtual network.
+    examples:
```

Comment:
Improve or remove example.

Reply:
I could see this staying since the `--ip-address` doesn't give the format.  But it could be removed since it only demonstrates supplying all required parameters.

Reply:
If `--ip-address` does not give a format, then that seems like a documentation failing for the parameter. In particular I bet users want to know if it accepts IPv6 as well as IPv4.